src/views/thingstotry/l10n-static.json

```diff
@@ -0,0 +1,24 @@
+{
+    "ttt.MakeItFlyActivityLoc": "/pdfs/cards/FlyCards.pdf",
+    "ttt.MakeItFlyGuideLoc": "/pdfs/guides/FlyGuide.pdf",
+    "ttt.AnimateYourNameActivityLoc": "/pdfs/cards/AnimateYourNameCards.pdf",
+    "ttt.AnimateYourNameGuideLoc": "/pdfs/guides/NameGuide.pdf",
+    "ttt.MakeMusicActivityLoc": "/pdfs/cards/MusicCards.pdf",
+    "ttt.MakeMusicGuideLoc": "/pdfs/guides/MusicGuide.pdf",
```

Comment:
I don't think the guides are in www – they're still in the old repo. I'll bring them over, just that these won't work until I do.